example/Jamfile

```diff
@@ -21,8 +22,8 @@ exe 05_a_overlay_polygon_example : 05_a_overlay_polygon_example.cpp ;
 exe 05_b_overlay_linestring_polygon_example : 05_b_overlay_linestring_polygon_example.cpp ;
 exe 06_a_transformation_example : 06_a_transformation_example.cpp ;
 exe 06_b_transformation_example : 06_b_transformation_example.cpp ;
-exe 07_a_graph_route_example : 07_a_graph_route_example.cpp ;
-exe 07_b_graph_route_example : 07_b_graph_route_example.cpp ;
+# exe 07_a_graph_route_example : 07_a_graph_route_example.cpp : <library>/boost/graph//boost_graph ;
+# exe 07_b_graph_route_example : 07_b_graph_route_example.cpp : <library>/boost/graph//boost_graph ;
```

Comment:
Indeed, boost graph is only used to show this possible collaboration

Reply:
I think it can be kept for now. It's not essential for testing or documentation.
We (I) should be able to fix it later after this rework has passed.